ports/meshoptimizer/vcpkg.json

```
@@ -1,7 +1,16 @@
 {
   "name": "meshoptimizer",
-  "version-string": "0.14",
-  "port-version": 2,
+  "version-string": "0.16",
```

```suggestion
  "version": "0.16",
```
This might be served by `version`.

---

Addressed in 3a728e9fab18a133f92a82a3d6e1218061399428.